src/backends/kafka/mappings.rs

```diff
@@ -0,0 +1,20 @@
+use crate::types::errors::KafkaErrorCode;
+use rdkafka::error::{KafkaError, RDKafkaErrorCode};
+
+use crate::backends::ConsumerError;
+
+fn map_error_code(code: RDKafkaErrorCode) -> KafkaErrorCode {
+    match code {
+        RDKafkaErrorCode::OffsetOutOfRange => KafkaErrorCode::OffsetOutOfRange,
+        _ => KafkaErrorCode::Unknown,
+    }
+}
```

Comment:
Do we *need* this? It's basically just 1:1. How about just passing the RdKafkaErrorCode straight to our `ConsumerError::OffsetFetch` without the map

Reply:
That makes our interface dependent on the rdkafka implementation of a Kafka library. Which means it makes the client dependent on such implementation

Reply:
Ok. Looking at this again, I guess it's because you did not want to expose RDKafka things externally. So, ignore what I said earlier: this seems fine to me.